ereg/registration.py

```diff
@@ -801,3 +797,34 @@ def _register_image_and_get_transform(
                 tmp.SetCenter(registration_transform_sitk.GetCenter())
                 registration_transform_sitk = tmp
         return registration_transform_sitk
+
+    def _setup_logger(self, log_file: str, logger_type: str) -> None:
+        """
+        Setup the logger.
+
+        Args:
+            log_file (str): The log file.
+            logger_type (str): The logger type.
+        """
+        reload(logging)
```

Comment:
this seems really weird. are you sure this is really needed? 
Maybe compare with the logger used in aurora ([here](https://github.com/BrainLesion/AURORA/blob/19c48955e2de54903e578c0f093edb72ecefbb85/brainles_aurora/inferer/inferer.py#L22)). However if it works reliably i guess its fine.

Reply:
that also puzzled me :)

Reply:
I don't have a lot of experience with the `logging` library (I rarely use it in my usual code, since all console messages either get saved by the HPC or get pushed to files). Anyway, I am happy to use whatever solution you propose that works for @neuronflow.